docs/stdlib/memory.rst

```diff
@@ -181,8 +181,6 @@ Simulation
 Memory description
 ==================
 
-.. autoexception:: amaranth.hdl.FrozenMemory
```

Comment:
This is now not documented at all, is it? How about adding `:noindex:` instead, so that you get the docstring here, but not the link target? Then some other place could be the canonical one for it.